webapp/src/app/models/map-styles.ts

```diff
@@ -1,12 +1,11 @@
-export type MapStyles = {
-	walls: WallColors;
-} & {
+export interface MapStyles {
 	[key: string]: MapStyle;
-};
+}
 
 export type MapStyle = {
 	sheet: string;
 	hasDoorMat?: boolean;
+	walls?: WallColors;
 } & {
 	[key: string]: any;
 };
```

Comment:
You might as well do this while you're at it
```suggestion
export interface MapStyle {
	sheet: string;
	hasDoorMat?: boolean;
	walls?: WallColors;
	[key: string]: any;
}
```